drafts/GoL-reules/Game of Links.md

```diff
@@ -0,0 +1,139 @@
+# The Game of Links. Goals, rules and provision.
+
+## Intro
+
+
+## General provision of the Game.
+
+![](euler-5_timeline.png)
```

Comment:
The structure is far from ideal. The need clear sequence of events including testing GoT and cyber•Auction tests...

Reply:
600k ATOMs is a goal now